docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -59,19 +59,30 @@ image::images/view-alert-details.png[Shows the Event Renderer button, 200]
 
 The Alert details flyout appears and offers several options for viewing alert details:
 
-* *Summary*: Shows an aggregated view of alert details. Alerts that have been enriched with `threat.indicator` data also display the *threat summary* section, which is an additional section located beneath the alert summary. In the *threat summary* section, you can view mapped data for the following `threat.indicator` subfields:
-** `matched.field`
-** `matched.type`
-** `source (threat.indicator.provider)`
-** `first_seen`
-** `last_seen`
+* *Summary*: Shows an aggregated view of alert details, plus two types of threat intelligence data: indicator rule enrichments and investigation time enrichments.
+** *Indicator rule enrichments*: As the name suggests, indicator rule enrichments only display for alerts produced by indicator match rules. You can spot an indicator rule enrichment by checking for fields pre-fixed by the {es-sec} icon and appended with source event data, if that information is available. Indicator rule enrichments are created when rule indicators are met and mapped index fields are produced. An alert can have one or more indicator rule enrichment and the same indicator match can display multiple times.
+** *Investigation time enrichments*: Displays for all alerts regardless of the rule type that generated it. Like indicator rule enrichments, investigation time enrichments are also pre-fixed by the {es-sec} icon as a way to set them apart from general alert details. Time enrichment data from the past 30 days is generated when the {es-sec} app queries the `filebeat-*` index for the following set of fields. If these field values aren't found, investigation time enrichments aren't created. Or, if an indicator match rule uses these fields for indicator mapping, and the indicator rule enrichment data matches the  investigation time enrichment data, the investigation time enrichments are treated as duplicated data and are removed. For example, if you specify `file.hash.sha256` and `file.hash.md5` as fields that should be mapped for your indicator match rule, these fields will only display as indicator rule enrichments on the **Summary** and **Threat Intel** tabs when the rule is activated and the alert is generated. You can learn more about the investigation time enrichment query by going to the **Threat Intel** tab and clicking the **Inspect** icon next to event you want to inspect.
```

Comment:
We could consider replacing the sentence `Time enrichment data from the past 30 days is generated when the {es-sec} app queries the filebeat-* index for the following set of fields` with something like `Indicator data from the past 30 days is queried to generate enrichments for the fields listed below.` and we can skip the next sentence. 

We can replace the sentence `Or, if an indicator match rule uses these fields for indicator mapping, and the indicator rule enrichment data matches the  investigation time enrichment data, the investigation time enrichments are treated as duplicated data and are removed` with something like `if investigation time matches return enrichments that are identical to indicator rule matches they will not appear twice`. 

In the next sentence we can say `are mapped` instead of `should be mapped`, and we can also say `when alerts are generated` to finish that sentence, as the rule can generate many alerts.